tools/buildmgr/test/testinput/Examples/GCC/Build_GPP/Simulation.cprj

```diff
@@ -19,6 +19,7 @@
   <target Ddsp="NO_DSP" Dendian="Little-endian" Dfpu="NO_FPU" Dmve="NO_MVE" Dname="ARMCM3" Dsecure="Non-secure" Dtz="NO_TZ" Dvendor="ARM:82">
     <output name="Build_G++" type="exe"/>
     <ldflags add="--entry=Reset_Handler --specs=nosys.specs -lstdc++ -mcpu=cortex-m3 -mthumb" compiler="GCC" file="./RTE/Device/ARMCM3/gcc_arm.ld"/>
+    <ldcxxflags add="-static" compiler="GCC"/>
```

Comment:
We are missing tests around scenarios mentioned [here](https://github.com/Open-CMSIS-Pack/devtools/issues/224#issuecomment-1274697707)

Reply:
Added [here](https://github.com/Open-CMSIS-Pack/devtools/pull/492/commits/73e7e7c04061bf4a1da262e693aa4a7204e6b561#diff-79dce1734fc53449a5cc900eb03c8984707c08804d1183b9d44d1becd45cce1a)